RecoEgamma/EgammaElectronAlgos/interface/TrajSeedMatcher.h

```diff
@@ -156,6 +185,32 @@ class TrajSeedMatcher {
     const std::vector<double> dRZMaxLowEtEtaBins_; 
     const std::vector<double> dRZMaxLowEt_; 
   };
+  class MatchingCutsV2 : public MatchingCuts {
+  public:
+    explicit MatchingCutsV2(const edm::ParameterSet& pset);
+    bool operator()(const SCHitMatch& scHitMatch)const;
+  private:
+    const egPM::Param<SCHitMatch> dPhiMin_;
+    const egPM::Param<SCHitMatch> dPhiMax_;
+    const egPM::Param<SCHitMatch> dRZMin_;
+    const egPM::Param<SCHitMatch> dRZMax_;
+  }; 
+
+  class MatchingCutsV3 : public MatchingCuts {
```

Comment:
I'd chose something more descriptive than V{1,2,3}, e.g.
MatchigCutsEt, MatchingCutsParam, MatchingCutsEtGrad.

feel free to ignore if there were good reasons for V{1,2,3}